client/blocks/stats-navigation/index.js

```diff
@@ -82,6 +95,7 @@ class StatsNavigation extends Component {
 				{ showIntervals && (
 					<Intervals selected={ interval } pathTemplate={ pathTemplate } standalone />
 				) }
+				{ ! stickers && siteId && <QueryBlogStickers blogId={ siteId } /> }
```

Comment:
If #28629 gets merged, you won't need to check `stickers` or `siteId` here. Then you also won't need to pass `stickers` as a prop to the component. Just use it for calculating `isWordAds` and nothing else.

Reply:
Today I Learned™️  about horizon - switched over to that instead of blog stickers.